### PR TITLE
[bitnami/minio]custom servicemonitor apiversion

### PR DESCRIPTION
**Pull request by lusson-luo**
Signed-off-by: lusson <lusson@foxmail.com>

### Description of the change

add values `metrics.serviceMonitor.apiVersion` to modify serviceMonitor api

### Benefits

To meet the needs of modifying the apiversion of serviceMonitor to access the monitoring platforms of some cloud manufacturers (such as Tencent Cloud Monitoring)

### Possible drawbacks

no

### Applicable issues

no

### Additional information

no

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/minio/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/minio
   - https://min.io
-version: 11.9.2
+version: 11.9.3
```
Comment (marcosbc) [inline bitnami/minio/Chart.yaml:28]: Since this is adding a new feature, could you increase the minor version? I.e. to `11.10.0`.

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please have a look at my comment whenever you have a chance.

File: bitnami/minio/Chart.yaml (hunk 1 above)
Comment (lusson-luo) [reply bitnami/minio/Chart.yaml:28]: ok

**Comment by lusson-luo** [comment]
> Thanks for the contribution! Please have a look at my comment whenever you have a chance.

i had upgrade version to 11.10.0
image

**Review by lusson-luo — COMMENTED** [review]
i hava review my changes

File: bitnami/minio/templates/servicemonitor.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.metrics.serviceMonitor.enabled }}
-apiVersion: monitoring.coreos.com/v1
+apiVersion: {{ .Values.metrics.serviceMonitor.apiVersion | default "monitoring.coreos.com/v1" }}
```
Comment (marcosbc) [inline bitnami/minio/templates/servicemonitor.yaml:2]: Minor improvement, for consistency with the rest of the codebase:
```suggestion
apiVersion: {{ default "monitoring.coreos.com/v1" .Values.metrics.serviceMonitor.apiVersion }}
```

File: bitnami/minio/values.yaml [hunk 3]
```diff
@@ -1041,6 +1041,8 @@ metrics:
     ## ref: https://github.com/bitnami/charts/tree/master/bitnami/prometheus-operator#prometheus-configuration
     ##
     selector: {}
+    ## @param metrics.serviceMonitor.apiVersion ApiVersion for the serviceMonitor Resource(defaults to "monitoring.coreos.com/v1")
```
Comment (marcosbc) [inline bitnami/minio/values.yaml:1044]: Minor wording improvement:
```suggestion
    ## @param metrics.serviceMonitor.apiVersion ApiVersion for the serviceMonitor Resource (defaults to "monitoring.coreos.com/v1")
```

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for making the changes! Could you have a look at a couple of small requests?

**Comment by marcosbc** [comment]
Could you also have a look to the [DCO](https://github.com/bitnami/charts/pull/12134/checks?check_run_id=8184662142)?

**Comment by lusson-luo** [comment]
o， 

> Could you also have a look into the [DCO](https://github.com/bitnami/charts/pull/12134/checks?check_run_id=8184662142)?

i'm sorry, i click Resolve conversation in github, it commit with Co-authored-by but no Signed-off-by. now i recommit with Signed-off-by.

**Review by marcosbc — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**